R/create.autoencoder.R

```diff
@@ -8,12 +8,12 @@ create.autoencoder <- function(
 	) {
 
 	# input checks
-	if (class(data.matrix) != 'matrix') {
+	if (class(data.matrix) != 'matrix' && class(data.matrix) != 'data.frame') {
 		stop('data.matrix needs to be a matrix');
```

Comment:
(question | blocking) I think `data.frame` works, right? Did you want to update the comment?